spec/cobra_commander/cli_spec.rb

```diff
@@ -21,12 +21,34 @@
         App
         ├── a
         │   ├── b
+        │   │   └── g
+        │   │       ├── e
+        │   │       └── f
         │   └── c
         │       └── b
-        └── d
+        │           └── g
+        │               ├── e
+        │               └── f
+        ├── d
+        │   ├── b
+        │   │   └── g
+        │   │       ├── e
+        │   │       └── f
+        │   └── c
+        │       └── b
+        │           └── g
+        │               ├── e
+        │               └── f
+        └── node_manifest
             ├── b
-            └── c
-                └── b
+        │       └── g
+        │           ├── e
+        │           └── f
```

Comment:
This is an existing bug with the Formatter. What ever needs to be done to resolve it should not be handled in this PR.